daemon/src/model/cfd.rs

```diff
@@ -1274,6 +1274,31 @@ impl Cfd {
             (None, None) => None,
         }
     }
+
+    /// Only cfds in state `Open` that have not received an attestation and are within 23 hours
+    /// until expiry are eligible for rollover
+    pub fn rollover_proposal(&self) -> Option<RollOverProposal> {
+        let now = OffsetDateTime::now_utc();
+        let remaining_till_expiry = self.expiry_timestamp() - now;
+        let rollover_window = Duration::hours(1);
+
+        if matches!(
```

Comment:
My personal preference is to do an early return instead of if/else, e.g. 

```
if !matches!(self.state.clone(),
            CfdState::Open {
                attestation: None,
                ..
            })
{
return None;
}
```

Reply:
Second this!

Reply:
this changed completely, I think you will like the change :)